app/controllers/claims_controller.rb

```diff
@@ -64,7 +64,8 @@ def claim_params
       :address_line_4,
       :postcode,
       :date_of_birth,
-      :teacher_reference_number
+      :teacher_reference_number,
+      :national_insurance_number
```

Comment:
We should probably get that trailing comma in to stop the red/green diff juggling. I wonder why Standard doesn't complain about this one? Because it's not explicitly wrapped in square brackets?